src/derive/command.rs

```diff
@@ -20,6 +25,25 @@ pub struct DeriveArgs {
 /// All possible subcommands for `ngs derive`.
 #[derive(Subcommand)]
 pub enum DeriveSubcommand {
+    /// Derives the quality score encoding used to produce the file.
+    Encoding(self::encoding::DeriveEncodingArgs),
+
+    /// Derives the endedness of the file.
+    Endedness(self::endedness::DeriveEndednessArgs),
+
     /// Derives the instrument used to produce the file.
     Instrument(self::instrument::DeriveInstrumentArgs),
+
+    /// Derives the read length of the file.
+    Readlen(self::readlen::DeriveReadlenArgs),
+
+    /// Derives the strandedness of the RNA-Seq file.
+    /// This subcommand requires a GFF file.
```

Comment:
You should put requirements like this here: just rely on the parameters being non-optional to state that it must be provided.

```suggestion
```